core/version.go

```diff
@@ -35,3 +36,25 @@ func ParseBlockVersion(protocolVersion string) (*semver.Version, error) {
 
 	return semver.New(versionVals[0], versionVals[1], versionVals[2], "", ""), nil
 }
+
```

Comment:
Add description to the `CheckBlockVersion` func
```suggestion

// CheckBlockVersion checks if the block protocol version is supported by Juno
```